contracts/USM.sol

```diff
@@ -29,13 +27,28 @@ contract USM is ERC20 {
         ethPool = 0;
     }
 
+    /**
```

Comment:
These functions are actually `integer * decimal = integer` operations, assuming that the decimal is in UNIT